packages/react/ds/src/checkbox/checkboxes-group.tsx

```diff
@@ -0,0 +1,68 @@
+import { ErrorText } from '../error-text/error-text.js';
+import { Heading, HeadingAs, HeadingSize } from '../heading/heading.js';
+import { HintText } from '../hint-text/hint-text.js';
+import Checkbox, { CheckboxSizeEnum } from './checkbox.js';
+
+type CheckboxesGroupType = {
+  size?: CheckboxSizeEnum;
+  fieldId: string;
+  errorMessage?: string;
+  items: {
+    value: string;
+    label?: string;
+    hint?: string;
+  }[];
+  title: {
+    value: string;
+    asHeading?: {
+      size: HeadingSize;
+      tag: HeadingAs;
+    };
+    hint?: string;
+  };
+};
+
+const CheckboxesGroup = ({
+  fieldId,
+  errorMessage,
+  title,
+  items,
+  size,
+}: CheckboxesGroupType) => {
+  return (
+    <div className="gi-flex">
+      {errorMessage && (
+        <div className="gi-w-5 gi-border-l-[5px] gi-border-l-red-600 gi-border-solid"></div>
+      )}
+      <fieldset>
+        <legend className="gi-mb-3.5 gi-font-primary sm:gi-text-sm md:gi-text-md lg:gi-text-lg">
```

Comment:
Question: I was reviewing the screenshot you included in the description, and I'm not sure if this will be addressed in the design review, but in the local HTML Storybook, there's noticeably less margin at the bottom after the legend.

Local HTML Checkbox:
![Screenshot 2024-09-24 at 10 11 08](https://github.com/user-attachments/assets/807ad241-c5d7-4195-a04a-6d1a699209c4)

Reply:
You mean the margin-bottom of the heading ? That comes from the `Heading` component. I think we need to review the `Heading` component to fix this

Reply:
Ok, we can then fix it in a separate PR, but lets file it somewhere on Azure 👍

Reply:
agreed